Misc/NEWS.d/next/Build/2024-07-16-11-18-43.gh-issue-120522.exdb3D.rst

```diff
@@ -0,0 +1,2 @@
+The app store compliance patch is now applied to the installed products,
+rather than to the original source tree.
```

Comment:
Since the original PR hasn't been in a release yet, there's no need to add a blurb for this change.